scripts/actions/remove-completed-batch.js

```diff
@@ -60,32 +60,38 @@ const removePageContext = async (fileUris) => {
       };
     });
 
-  return contextUids.reduce(async (returnCode, { contextUid, fileUri }) => {
-    const url = new URL(
-      `/context-api/v2/projects/${PROJECT_ID}/contexts/${contextUid}`,
-      process.env.TRANSLATION_VENDOR_API_URL
-    );
+  const results = await Promise.all(
+    contextUids.map(async ({ contextUid, fileUri }) => {
+      const url = new URL(
+        `/context-api/v2/projects/${PROJECT_ID}/contexts/${contextUid}`,
+        process.env.TRANSLATION_VENDOR_API_URL
+      );
+
+      const options = {
+        method: 'DELETE',
+        headers: {
+          Authorization: `Bearer ${accessToken}`,
+        },
+      };
 
-    const options = {
-      method: 'DELETE',
-      headers: {
-        Authorization: `Bearer ${accessToken}`,
-      },
-    };
+      const resp = await fetch(url.href, options);
 
-    const resp = await fetch(url.href, options);
+      const { response } = await resp.json();
+      const { code } = response;
 
-    const { response } = await resp.json();
-    const { code } = response;
+      return { code, fileUri };
+    })
+  );
 
-    if (code === 'SUCCESS' && resp.ok) {
+  return results.reduce(({ code, fileUri }) => {
+    if (code === 'SUCCESS') {
       console.log(`[*] Successfully deleted ${fileUri} context.`);
     } else {
       console.error(`[!] Unable to delete ${fileUri} context.`);
       return code;
     }
     return returnCode;
```

Comment:
```suggestion
    if (code === 'SUCCESS') {
      console.log(`[*] Successfully deleted ${fileUri} context.`);
      return returnCode;
    } else {
      console.error(`[!] Unable to delete ${fileUri} context.`);
      return code;
    }
```

[nit] Since you're using an `if`/`else` here, it might be best to return the `returnCode` in the `if` block, since thats the only time line 93 will be run: